codespell_lib/data/dictionary.txt

```diff
@@ -34091,13 +34451,16 @@ thann->than, thank,
 thansk->thanks
 thansparent->transparent
 thant->than
+thar->than, that,
 thare->there
 thast->that, that's,
 that;s->that's
 thatn->that, than,
 thats'->that's
 thats->that's
 thats;->that's
+thaught->thought
```

Comment:
`thaught->thought, taught,`?
`thaughts->thoughts, taughts,`?

Reply:
> `thaught->thought, taught,`?

Added.

> `thaughts->thoughts, taughts,`?

I'm not sure about `taughts`. Is it a correct word?

Reply:
No :blush:

Reply:
> No :blush: 

:smile:  :+1: